idc/views.py

```diff
@@ -240,6 +240,25 @@ def help_page(request):
 def quota_page(request):
     return render(request, 'idc/quota.html', {'request': request, 'quota': settings.IMG_QUOTA})
 
+@login_required
+def fetch_cohort_data(request):
```

Comment:
Not clear on what this view does?

Reply:
An aborted refactor. Sorry I missed this. I just deleted it.